### PR TITLE
<Carousel/> - Add controlsPosition prop

### PR DESCRIPTION
**Pull request by EgoziE**
In this PR:
1. Refactor story page
2. Wrote nice documentation to Testkit and examples
3. Added controlsPosition prop:
image

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6164

File: src/Carousel/index.d.ts [hunk 1]
```diff
@@ -1,5 +1,13 @@
 import * as React from 'react';
 
+export type CarouselControlsPosition = 'sides' | 'overlay' | 'bottom' | 'none';
+
+export default class Carousel extends React.Component<CarouselProps> {}
+
+export type CarouselButtonSkin = 'standard' | 'inverted';
+
+export type CarouselImage = Partial<HTMLImageElement> & { src: string };
```
Comment (gendeld) [inline src/Carousel/index.d.ts:9]:
```suggestion
export type CarouselImage = OmitPolyfill<HTMLImageElement, 'src'> & { src: string };
```

**Comment by gendeld** [comment]
Maybe add `controlsPosition` to `exampleProps` ?

**Review by gendeld — APPROVED** [review]

File: src/Carousel/index.d.ts (hunk 1 above)
Comment (EgoziE) [reply src/Carousel/index.d.ts:9]: I tried it, but got errors, reverted.

**Pull request merged**